pyth-sdk-solana/src/state.rs

```diff
@@ -339,77 +416,96 @@ pub struct PriceAccount {
     pub comp:           [PriceComp; 32],
 }
 
+price_account_impl!(PriceAccountSolana);
+
 #[cfg(target_endian = "little")]
-unsafe impl Zeroable for PriceAccount {
+unsafe impl Zeroable for PriceAccountSolana {
```

Comment:
It's fine not to have this for `PriceAccountPythnet` because there's no function to load `PriceAccountPythnet` from this sdk